src/overlays/actors/ovl_En_Bigslime/z_en_bigslime.c

```diff
@@ -153,7 +153,7 @@ static Vtx sBigslimeTargetVtx[BIGSLIME_NUM_VTX] = {
  * Purpose: indices to calculate/update normal vector for lighting calculations
  * "EnBigslimeTri" struct based on "Tri" struct from gbi.h, but without the flag
  */
-static EnBigslimeTri sEnbigslimeTri[BIGSLIME_NUM_FACES] = {
+static EnBigslimeTri sEnBigslimeTri[BIGSLIME_NUM_FACES] = {
```

Comment:
```suggestion
static BigslimeTri sBigslimeTri[BIGSLIME_NUM_FACES] = {
```
Nice cleanup of the Bigslime xml, cool that it works now without having multiple files explicitly named. Since you're cleaning this up, this is probably fine? To be consistent with the xml names?